aiida/common/hashing.py

```diff
@@ -288,5 +288,7 @@ def float_to_text(value, sig):
     :param value: the float value to convert
     :param sig: choose how many digits after the comma should be output
     """
+    if value == 0:
```

Comment:
Maybe we should add a comment saying why this is necessary. Just looking at the code, it seems superfluous.